src/PawsyModules/Meowboard/Meowboard.cs

```diff
@@ -246,11 +249,11 @@ protected class TreasureHunter(MeowBoardModule Owner)
         protected WeakReference<MeowBoardModule> Owner = new(Owner);
         protected ConcurrentBag<ulong> TreasureHunters = [];
         protected ulong FirstResponder = 0;
-        protected bool GameActive = false;
+        public bool GameActive = false;
```

Comment:
I'd like it if you could make this and the other public field `internal` if possible